pytorch_lightning/loops/fit_loop.py

```diff
@@ -104,7 +117,13 @@ def max_steps(self) -> int:
     def max_steps(self, value: int) -> None:
         """Sets the maximum number of steps (forwards to epoch_loop)"""
         # TODO(@awaelchli): This setter is required by debugging connector (fast dev run), should be avoided
-        if value and value < -1:
+        if value is None:
+            rank_zero_deprecation(
```

Comment:
A deprecation test is missing

Check the tests in https://github.com/PyTorchLightning/pytorch-lightning/blob/master/tests/deprecated_api/test_remove_1-7.py